docs/demoprojects/api/DemoProject/DemoClass/methods/Method9.md

```diff
@@ -14,7 +14,7 @@
 Example of an method with a optional parameters.
 
 ```csharp
-public void Method9(string value1 = "someString", string value2 = null);
+public void Method9(string value1 = "someString", string value2 = null, decimal? value3 = -792281625142643.37593543950335, decimal? value4 = null, CancellationToken value5 = default);
```

Comment:
Something to not here: we still aren't including the `M` suffix which means technically you cannot copy this definition into code and have it compile. I tend to think that's OK because readability doesn't suffer at all (and the issue already exists for `float`), but I wanted to bring it up regardless.

Reply:
Nice catch, I've added the `f` and `M` suffixes with 5fb14734860a3a7f0cc0d3f1f6b1cd484ae10941